scripts/precompute.py

```diff
@@ -0,0 +1,60 @@
+# precompute detection with YOLOX detection
```

Comment:
I think it is better to move this file to test directory? This scripts is only used for generate artifacts for the tests. @yushan111 what do you think?

Reply:
Folder structures are not final yet. But yep, could be moved to `test/scripts`

Reply:
> I think it is better to move this file to test directory? This scripts is only used for generate artifacts for the tests. @yushan111 what do you think?

Agree with that. Thanks for the careful review!